### ISSUE TITLE
Is this Project Dead?

### ISSUE THREAD

**Issue by douglagm** [SUSE/Portus#2352]
Hi,

I see there has been very little to no activity on the Project lately...

Is this project dead?

Regards

**Comment by RobertDWhite** [comment]
It does seem to be, to me.

**Comment by scherer-michael** [comment]
You may want to have a look at this project : https://github.com/goharbor/harbor
But, maybe, way more complex than what you are searching...

**Comment by bclermont** [comment]
all ruby on rails projects must die. slow, bloated, complex code, way too much magic syntax. impossible to maintains. 

harbor is in golang, even if vmware completely ignore their community, it's better.

**Comment by RobertDWhite** [comment]
Probably useless, but this is something I came up with for an on-prem installation of Harbor for internal repo things. If nothing else, hopefully it can get you started quickly.
https://github.com/RobertDWhite/harbor-registry

**Comment by Vad1mo** [comment]
Yes, this project is dead. The principal contributors to the project aren't on the project anymore or left the company. I reached out to SUSE a few years ago to step in as a contributor, but they didn't even reply. In retrospect: lucky me. We ended up switching to Harbor, so did SUSE. See https://www.suse.com/c/suse-private-registry-harbor/
